tests/python_functional/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py26,py27,py36,py37,py38
+skipsdist = True
+[testenv]
+deps =
+    pytest
+    psutil
+    pathlib2
+passenv = INSTALL_DIR
+commands =
+    pytest --installdir={env:INSTALL_DIR:} functional_tests/ src/
```

Comment:
For some reason all app-parser tests fail for me with `Exception: syslog-ng can not started`. May I ask if these pass for you? Other tests run correctly though.

Reply:
As I wrote in my first comment you should export the INSTALL_DIR variable before the run.
$ export INSTALL_DIR=[your syslog-ng install path] 
$ tox

Reply:
My bad sorry! I compiled syslog-ng without app-model support.